lms/static/js/learner_dashboard/views/certificate_view.js

```diff
@@ -0,0 +1,33 @@
+;(function (define) {
+    'use strict';
+    define(['backbone',
+            'jquery',
+            'underscore',
+            'gettext',
+            'text!../../../templates/learner_dashboard/certificate.underscore'
+           ],
+         function(
+             Backbone,
+             $,
+             _,
+             gettext,
+             certificateTpl
+         ) {
+            return Backbone.View.extend({
+                el: '.certificates-list',
+                tpl: _.template(certificateTpl),
+                initialize: function(data) {
+                    this.context = data.context;
+                    this.certificatesData = this.context.certificatesData;
+                    this.xseriesImage = this.context.xseriesImage;
+                    this.render();
+                },
+                render: function() {
+                    if (this.certificatesData.length > 0) {
+                        this.$el.html(this.tpl({certificates: this.certificatesData, image: this.xseriesImage}));
```

Comment:
You can just pass this.context to the template, and the template can look for the certificatesData and xseriesImage keys. It is unnecessary to create separate variables for these values.